Source/Provers/SMTLib/SMTLibBatchTheoremProver.cs

```diff
@@ -163,6 +163,9 @@ public override async Task<Outcome> CheckOutcome(ErrorHandler handler, int error
     private async Task<Outcome> GetResponse(CancellationToken cancellationToken)
     {
       var outcomeSExp = await Process.GetProverResponse().WaitAsync(cancellationToken);
+      if (outcomeSExp.Name.Equals("timeout")) {
```

Comment:
Can we have a unit test for this?

Reply:
Yeah, we can. I should have left this as a draft until I added one. :)